sqldb/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "sqldb"
+version = "0.1.0"
+description = "Postgres SQL Database capability provider for contract wasmcloud:sqldb"
+edition = "2018"
+resolver = "2"
+
+[dependencies]
+async-trait = "0.1"
+base64 = "0.13"
+bytes = "1.0"
+chrono = "0.4"
+crossbeam="0.8"
+futures = "0.3"
+log = "0.4"
+minicbor = { version = "0.11", features = ["derive", "std", "half"] }
+once_cell = "1.8"
+rmp-serde = "0.15"
+serde_bytes = "0.11"
+serde_json = "1.0"
+serde = { version = "1.0", features = ["derive"] }
+thiserror = "1.0"
+tokio = { version = "1", features = ["full"] }
+tokio-postgres = { version = "0.7", features=["with-bit-vec-0_6", "with-chrono-0_4","with-uuid-0_8", "with-eui48-0_4"]}
+bb8-postgres = "0.7"
+bit-vec = "0.6"
+uuid = "0.8"
+#postgres-native-tls = "0.5"
+native-tls = "0.2"
+toml = "0.5"
+wasmcloud-interface-sqldb = { path="../../interfaces/sqldb/rust" }
```

Comment:
Assuming this will be swapped with a crates reference when available.

Reply:
after it's published to crates.io